tensorflow/python/ops/rnn_cell_impl.py

```diff
@@ -289,6 +288,86 @@ def __call__(self, inputs, state, scope=None):
     # method.  See the class docstring for more details.
     return base_layer.Layer.__call__(self, inputs, state, scope=scope)
 
+class SRUCell(_LayerRNNCell):
+  """Training RNNs as Fast as CNNs (cf. https://arxiv.org/abs/1709.02755).
```

Comment:
Drive-by comment: Can you expand on the description here? This seems a bit terse.

Things to include:
- What does "SRU" stand for :)
- A description of the basic characteristics of this cell.